tests/console/ndctl.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: install ndctl. Destroy and configure NVDIMM namespaces
+# Maintainer: Alvaro Carvajal <acarvajal@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils qw(zypper_call);
+
+sub run {
+    my $self = shift;
+    return unless get_var('NVDIMM');
+    $self->select_serial_terminal;
+    zypper_call('in ndctl');
+    assert_script_run 'ndctl destroy-namespace --force all';
+    my $total = get_var('NVDIMM_NAMESPACES_TOTAL', 2);
+    foreach my $i (0 .. ($total - 1)) {
+        my $device = script_output 'ndctl create-namespace --mode=fsdax --map=dev';
```

Comment:
I assume we would need something like 

`ndctl create-namespace --force --reconfig=namespace0.0 --mode=memory --map=dev`

Reply:
I am destroying all namespaces in L24, so I do not think `--reconfig` is necessary in this case. We could extend the module so it does reconfigure instead of destroy+create-namespace, but would that be useful for the test?

With regards to `--mode=memory`, I did not find that option in the ndctl-create-namespace manpage. Only raw, sector, fsdax or devdax.

Reply:
Tested with `--mode=memory` successfully, so I changed the call. Did not add `--reconfig` as the namespaces are destroyed in L24.